rsconnect/main.py

```diff
@@ -1443,6 +1492,9 @@ def write_manifest_notebook(
     file,
     extra_files,
     image,
+    disable_env_management,
```

Comment:
Checking my understanding: this var isn't actually needed in the function body because the callback that consumes it has already been called before line 1501?

Reply:
You're right that it's only required by the callback function. `click` will complain if it doesn't know how to invoke `write_manifest_notebook` which is why it needs to be included in the function signature even though it isn't used.

```
TypeError: write_manifest_notebook() got an unexpected keyword argument 'disable_env_management'
```